prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -660,14 +660,14 @@ abstract class CoolTest {
                     val expected = if (action == TaskAction.receive && asyncProcessMode) {
                         totalItems
                     } else totalItems / receivers.size
-                    if (count == null || expected != count) {
-                        queryResults += Pair(
+                    queryResults += if (count == null || expected != count) {
```

Comment:
Yeah, I like this porting code refactoring.

Reply:
I concur